WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -204,19 +224,100 @@ private extension GetStartedViewController {
     
 }
 
-// MARK: - UITableViewDataSource
+// MARK: - Social Button Management
 
-extension GetStartedViewController: UITableViewDataSource {
+private extension GetStartedViewController {
     
-    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return rows.count
+    func configureSocialButtons() {
+        guard let buttonViewController = buttonViewController else {
+            return
+        }
+        
+        buttonViewController.hideShadowView()
+        
+        if WordPressAuthenticator.shared.configuration.enableSignInWithApple {
+            if #available(iOS 13.0, *) {
+                buttonViewController.setupTopButtonFor(socialService: .apple, onTap: appleTapped)
+            }
+        }
+        
+        buttonViewController.setupButtomButtonFor(socialService: .google, onTap: googleTapped)
+        
+        let termsButton = WPStyleGuide.signupTermsButton()
+        buttonViewController.stackView?.addArrangedSubview(termsButton)
+        termsButton.addTarget(self, action: #selector(termsTapped), for: .touchUpInside)
     }
+    
+    @objc func appleTapped() {
+        tracker.set(flow: .loginWithApple)
+        tracker.track(click: .loginWithApple)
+        
+        AppleAuthenticator.sharedInstance.delegate = self
+        AppleAuthenticator.sharedInstance.showFrom(viewController: self)
+    }
+    
+    @objc func googleTapped() {
+        tracker.set(flow: .loginWithGoogle)
+        tracker.track(click: .loginWithGoogle)
```

Comment:
should we track these after the `guard let..` or do we want to track this tap regardless if `GoogleAuthViewController.instantiate` fails or not?

Reply:
Hey @Gio2018 . That's a good question. I've seen most of the tracks happen before guards, so I assume tracking is meant to happen no matter what. I need to review tracking on this page anyway (this bit was just copied from another VC) so I'll bring that up with the team and change it if necessary. Thanks!!

Reply:
Hey @Gio2018 . Well that was an easy discussion. We agreed tap events should be tracked regardless, so they should be before guards, etc. And I just noticed I put the `termsOfService` track in the wrong place. Thanks for bringing this up!